### PR TITLE
Remove a message that tells whether a user exists

### PR DESCRIPTION
**Pull request by msimkunas**
This PR removes a message that tells the client whether a user with a given email address exists on the system.

### CODE REVIEW

**Comment by bennothommo** [comment]
@msimkunas I'm assuming this is to stop people from being able to work out which accounts exist on a site or not (user enumeration).

Does this change result in no flash message being shown if the user does not exist? If there's a flash message shown if the user does exist, this will just provide another avenue of enumeration - watch to see if a flash message is thrown.

Ideally, we should be throwing a flash message no matter if the user exists or not, saying something along the lines of "If your account exists, you will receive an email shortly with instructions on resetting your password". This prevents people from being able to work out if the user exists based on the message provided. (Ideally, we should also be queueing the email sent as well so that people can't use timing to work out if the user exists or not, based on the delay of an email being sent).

**Comment by msimkunas** [comment]
@bennothommo I've updated the partial text and changed `Mail::send` to `Mail::queue` so that emails are queued.

**Comment by LukeTowers** [comment]
@bennothommo LGTM, will let you do the merge once you're happy :)

File: components/ResetPassword.php [hunk 1]
```diff
@@ -90,7 +89,7 @@ public function onRestorePassword()
             'code' => $code
         ];
 
-        Mail::send('winter.user::mail.restore', $data, function($message) use ($user) {
+        Mail::queue('winter.user::mail.restore', $data, function($message) use ($user) {
```
Comment (AIC-BV) [inline components/ResetPassword.php:92]: What if no queue is set up?

Comment (msimkunas) [reply components/ResetPassword.php:92]: It should default to the sync driver: https://github.com/wintercms/winter/blob/develop/config/queue.php#L16

...unless it has been overridden for some reason. @LukeTowers @bennothommo should a separate notice be issued in the release notes?

Should we handle cases where a project uses no queues but still has the QUEUE_CONNECTION variable set to something other than sync?

Comment (bennothommo) [reply components/ResetPassword.php:92]: Yeah we can certainly add a note to the README that states that mail from this plugin will be queued.

**Pull request merged**